pkgs/development/compilers/yosys/plugins/synlig.nix

```diff
@@ -97,5 +97,6 @@ stdenv.mkDerivation (finalAttrs: {
     license     = licenses.asl20;
     maintainers = with maintainers; [ hzeller ];
     platforms   = platforms.all;
+    broken      = true; # Does not compile with yosys 0.39
```

Comment:
In #318592, I narrowed this to be more explicit with the specific version at hand, but this is also good.